packages/react-liveness/src/components/FaceLivenessDetector/shared/Hint.tsx

```diff
@@ -157,9 +157,8 @@ export const Hint: React.FC<HintProps> = ({ hintDisplayText }) => {
   }
 
   if (isRecording && !isFlashingFreshness) {
-    // During face matching, we want to only show the TOO_CLOSE or
-    // TOO_FAR texts. If FaceMatchState matches TOO_CLOSE, we'll show
-    // the TOO_CLOSE text, but for FACE_IDENTIFED, CANT_IDENTIFY, TOO_MANY
+    // During face matching, we want to only show the
```

Comment:
Doesn't TOO_CLOSE still show? I see a conditional below for `FaceMatchState.TOO_CLOSE`

Reply:
Oh good catch, it doesn't in that case because there is no TOO_CLOSE state anymore. But the "Move back" text would show up [here](https://github.com/aws-amplify/amplify-ui/pull/5085/files#diff-af985946f88a3ee83d7df8cbb5644c899f1c4d41106ed01c7b1dd018d3d5d626L128) it just evaluates before recording and doesn't rely on the TOO_CLOSE state.